src/3d/framegraph/qgsdebugtexturerenderview.h

```diff
@@ -0,0 +1,64 @@
+/***************************************************************************
+  qgsdebugtexturerenderview.h
+  --------------------------------------
+  Date                 : June 2024
+  Copyright            : (C) 2024 by Benoit De Mezzo
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSDEBUGTEXTURERENDERVIEW_H
+#define QGSDEBUGTEXTURERENDERVIEW_H
+
+#include "qgsabstractrenderview.h"
+
+namespace Qt3DRender
+{
+  class QRenderSettings;
+  class QLayer;
+  class QSubtreeEnabler;
+  class QTexture2D;
+  class QCamera;
+  class QCameraSelector;
+  class QLayerFilter;
+  class QRenderTargetSelector;
+  class QClearBuffers;
+  class QRenderStateSet;
```

Comment:
only QLayer forward declaration should be needed here...

Reply:
fixed!